### DISCUSSION TITLE
Is it possible for scvelo to get the correct result from cells with 3 day interval

### DISCUSSION THREAD

**Discussion by BW15061999** [theislab/scvelo#909] [Q&A] [answered]
Hi,

I have a dataset with 4.5d and 7d cells, but the direction inferred from scvelo is from 7d to 4.5d. I am not sure about this, because most of the 7d cells are memory cells, which are quite possible for them to differentiate back to 4.5d effector cells in real stiuation.

Best

**Comment by WeilerP** [comment, marked as answer]
@BW15061999, please have a look at related discussions (e.g. #216, #462, or #903).
